lib/phoenix_swagger/conn_validator.ex

```diff
@@ -35,56 +36,70 @@ defmodule PhoenixSwagger.ConnValidator do
   defp validate_boolean(_name, value, parameters) when value in ["true", "false"] do
     validate_query_params(parameters)
   end
+
   defp validate_boolean(name, _value, _parameters) do
     {:error, "Type mismatch. Expected Boolean but got String.", "#/#{name}"}
   end
 
   defp validate_integer(name, value, parameters) do
     _ = String.to_integer(value)
     validate_query_params(parameters)
-  rescue ArgumentError ->
+  rescue
+    ArgumentError ->
       {:error, "Type mismatch. Expected Integer but got String.", "#/#{name}"}
   end
 
   defp validate_number(name, value, parameters) do
     {_, ""} = Float.parse(value)
     validate_query_params(parameters)
-  rescue MatchError ->
+  rescue
+    MatchError ->
       {:error, "Type mismatch. Expected Number but got String.", "#/#{name}"}
   end
 
   defp validate_query_params([]), do: :ok
+
   defp validate_query_params([{_type, _name, nil, false} | parameters]) do
     validate_query_params(parameters)
   end
+
   defp validate_query_params([{_type, name, nil, true} | _]) do
     {:error, "Required property #{name} was not present.", "#"}
   end
+
   defp validate_query_params([{"string", _name, _val, _} | parameters]) do
     validate_query_params(parameters)
   end
+
   defp validate_query_params([{"integer", name, val, _} | parameters]) do
     validate_integer(name, val, parameters)
   end
+
   defp validate_query_params([{"number", name, val, _} | parameters]) do
     validate_number(name, val, parameters)
   end
+
   defp validate_query_params([{"boolean", name, val, _} | parameters]) do
     validate_boolean(name, val, parameters)
   end
+
   defp validate_query_params(path, conn) do
     [{_path, _basePath, schema}] = :ets.lookup(@table, path)
+
     parameters =
       for parameter <- schema.schema["parameters"],
           parameter["type"] != nil,
           parameter["in"] in ["query", "path"] do
-        {parameter["type"], parameter["name"], get_param_value(conn.params, parameter["name"]), parameter["required"]}
+        {parameter["type"], parameter["name"], get_param_value(conn.params, parameter["name"]),
```

Comment:
The line break here feels arbitrary. Maybe split it into 1 tuple element per line or introduce some local variables so the tuple fits on a single line again.